content/source/docs/enterprise/logging/index.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "enterprise2"
+page_title: "Terraform Enterprise Logging"
+sidebar_current: "docs-enterprise2-home"
+---
+
+## Terraform Enterprise Logs
```

Comment:
Formatting quibble: Should decrease each header level by one now that this is a page. So `#` here, `##` for the next one, etc.